### DISCUSSION TITLE
I can't figure out how to check plots on linux with the ubuntu installer

### DISCUSSION THREAD

**Discussion by redwoodimage** [Chia-Network/chia-blockchain#6669] [Support]
when using the .deb package i can't use the chia plots check -h, command 
so i tried chia-blockchain  plots check -h and that didn't work. After some googling the only info i found was uninstall it and install the cmd line version.

**Comment by lvcivs** [comment]
Try this:

1. `cd /usr/lib/chia-blockchain/resources/app.asar.unpacked/daemon/`
2.   ` ./chia plots check`

**Comment by BrandtH22** [comment]
@redwoodimage , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
